modules/libnetconf2-netconf-server@2023-09-07.yang

```diff
@@ -265,35 +265,33 @@ module libnetconf2-netconf-server {
       "Grouping for the SSH Keyboard interactive authentication method.";
 
     container keyboard-interactive {
-      presence "Indicates that PAM configuration file name has been configured.
-               This statement is present so the mandatory descendant
-               nodes do not imply that this node must be
-               configured.";
+      presence "Indicates that PAM configuration file name has been configured and that
+                the given client supportsthe SSH Keyboard Interactive authentication method.";
       description
         "Keyboard interactive SSH authentication method.";
-      leaf pam-config-file-name {
-        type string;
-        mandatory true;
-      }
-      leaf pam-config-file-dir {
-        type string;
-      }
+
+      reference
+        "RFC 4256:
```

Comment:
This should probably point to a page in libnetconf2's documentation online which explains how the authentication is handled.